pkg/apis/flowcontrol/validation/validation.go

```diff
@@ -303,6 +303,20 @@ func ValidateFlowSchemaResourcePolicyRule(rule *flowcontrol.ResourcePolicyRule,
 		}
 	}
 
+	if len(rule.ResourceNames) == 0 {
+		allErrs = append(allErrs, field.Required(fldPath.Child("resourceNames"), `resource rules must supply at least one resource names`))
```

Comment:
the English text should be "at least one resource name".

Reply:
updated